relay-server/src/utils/api.rs

```diff
@@ -35,8 +47,13 @@ impl ApiErrorResponse {
         ApiErrorResponse {
             detail: Some(messages.remove(0)),
             causes: messages,
+            relay: None,
         }
     }
+
+    pub fn relay_action(&self) -> Option<RelayErrorAction> {
```

Comment:
_nit_: You can create a default `None` action, and then just return `RelayErrorAction::None` by default.

Reply:
good idea